tests/state_test.go

```diff
@@ -713,6 +713,23 @@ func TestAllETH(t *testing.T) {
 	// EIP 158/161 skipped tests
 	skipTests["RevertPrefoundEmptyOOG.json"] = "State trie clearing unimplemented"
 	skipTests["FailedCreateRevertsDeletion.json"] = "State trie clearing unimplemented"
+	
+	//Create2 related skipped tests, all these tests use Create2 opcode
```

Comment:
Should be able to remove these now, if they are related to Create2, since that is in now